modules/hosted-control-planes-overview.adoc

```diff
@@ -0,0 +1,37 @@
+= Overview of hosted control planes (Technology Preview)
+
+You can use hosted control planes for Red Hat {product-title} to reduce management costs, optimize cluster deployment time, and separate management and workload concerns so that you can focus on your applications.
+
+You can enable hosted control planes as a Technology Preview feature through the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/multicluster_engine/requirements-and-recommendations[multicluster engine for Kubernetes operator version 2.0] on Amazon Web Services (AWS).
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+== Architecture of hosted control planes
+
+Typically, {product-title} is deployed in a coupled model, where a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane is the hot path, including compute, storage, and networking where workloads and applications run. The control plane is hosted by a dedicated group of nodes, which can be physical or virtual, with a minimum number to ensure quorum. The network stack is shared, and admin access to a cluster offers visibility into the cluster's control plane, machine management APIs, and other components that contribute to the state of a cluster.
+
+While the coupled model works well, some situations require an architecture where the control plane and data plane are decoupled. In those cases, the data plane is on a separate network domain with a dedicated physical hosting environment. The control plane is hosted by using high-level primitives such as deployments and stateful sets that are native to Kubernetes. That is, the control plane is treated as any other workload.
+
+image::hosted-control-planes-diagram.png[Diagram that compares the hosted control plane model against OpenShift with a coupled control plane and workers]
+
+== Benefits of hosted control planes
+
+With hosted control planes for {product-title}, you can pave the way for a true hybrid-cloud approach and enjoy several other benefits.
+
+* The security boundaries between management and workloads are stronger because the control plane is decoupled and hosted on a dedicated hosting service cluster. As a result, you are less likely to leak credentials for clusters to other users. Because infrastructure secret account management is also decoupled, cluster infrastructure administrators cannot accidentally delete control plane infrastructure.
+
+* With hosted control planes, you can run many control planes on fewer nodes. As a result, clusters are more affordable.
+
+* Because the control planes consist of pods that are launched on {product-title}, control planes start quickly. In addition, because the control plane is another Kubernetes workload, the same principles apply, such as monitoring, logging, and auto-scaling.
```

Comment:
I feel like _in addition_ and _the same_ are doing the same job. And I think you have made the case for the CP running as a workload in the first sentence. WDYT of a bit of minimalism in the second sentence?
```suggestion
* Because the control planes consist of pods that are launched on {product-title}, control planes start quickly. The same principles apply to <processes/workloads> such as monitoring, logging, and auto-scaling.
```